src/output/details.rs

```diff
@@ -185,11 +189,23 @@ impl<'a> Render<'a> {
         Ok(())
     }
 
+    /// Whether to show the extended attribute hint
+    pub fn show_xattr_hint(&self, file: &File<'_>) -> bool {
+        // Do not show the hint '@' if the only extended attribute is the security
+        // attribute and the security attribute column is active.
+        let xattr_count = file.extended_attributes.len();
+        let has_selinux_ctx = match file.security_context().context {
+            SecurityContextType::SELinux(_) => true,
+            SecurityContextType::None       => false,
+        };
+
+        xattr_count > 1 || (xattr_count == 1 && (!has_selinux_ctx || !self.opts.secattr))
```

Comment:
I find this condition very hard to read.

```suggestion
        let selinux_ctx_shown = has_selinux_ctx && self.opts.secattr;

        xattr_count > 1 || (xattr_count == 1 && !selinux_ctx_shown)
```